artifactory/commands/mvn/mvn.go

```diff
@@ -141,7 +141,8 @@ func (mc *MvnCommand) Run() error {
 		SetBuildConf(mc.configuration).
 		SetGoals(mc.goals).
 		SetInsecureTls(mc.insecureTls).
-		SetDisableDeploy(mc.deploymentDisabled)
+		SetDisableDeploy(mc.deploymentDisabled).
+		SetThreads(mc.threads)
```

Comment:
What parameter this will set on the mc object?

Reply:
publish.forkCount
@galusben

Reply:
we already support this param but here we just missed the internal threads param passing